modules/tests/shared/src/test/scala/lucuma/core/util/CalculatedValueSuite.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.util
+
+import cats.Invariant
+import cats.Monad
+import cats.kernel.laws.discipline.*
+import cats.laws.discipline.*
+import lucuma.core.util.arb.ArbCalculatedValue.given
+import munit.*
+
+class CalculatedValueSuite extends DisciplineSuite:
+  checkAll("CalculatedValue.Monoid", MonoidTests[CalculatedValue[Int]].monoid)
+
+  // Without explicitly defining which version to use, we get this error:
+  //    But both object given_Monad_CalculatedValue in object CalculationValue
+  //    and object given_Traverse_CalculatedValue in object CalculationValue
+  //    match type cats.Invariant[lucuma.core.util.CalculatedValue].
+  // How do you fix this in a less clumsy way?
+  given Invariant[CalculatedValue] = Monad[CalculatedValue]
+  checkAll("CalculatedValue.Monad",  MonadTests[CalculatedValue].monad[Int, String, Long])
```

Comment:
That's odd. I don't know who is asking for `Invariant`. But what you have here works. The alternative would be passing the correct instance explicitly, if you can figure out where it's demanded.

Reply:
Defining it as 

```scala
given Monad[CalculatedValue] with Traverse[CalculatedValue] with
  ...
```

(instead of separately) solves the problem.  Thanks @toddburnside for the tip.

Reply:
Aha, duh. 👍🏻